app/javascript/.eslintrc.js

```diff
@@ -40,6 +40,7 @@ module.exports = {
   },
   plugins: ['import', 'react', 'jsx-a11y'],
   rules: {
+    'import/export': ['warn'],
```

Comment:
I'm looking to fix this in another PR. Not sure yet if I can do better than this though.